hypothesize/utilities.py

```diff
@@ -2261,32 +2262,69 @@ def linhat(x, con, est, *args):
 
     return psihat
 
-def mcpKadjp(p,proc="Holm"):
+def mcpKadjp(p, k=1, proc = 'Holm'):
+    '''
+    MCP method based on results in Keselman, H. J., Miller, C. E., & Holland, 
+    B. (2011). Many tests of significance: New methods for controlling Type 
+    I errors. Psychological Methods, 16, 420-431.
+
+    Also see
+    Keselman, H. J., & Miller, C. E. (2012). 
+    Correction to many tests of significance: 
+    New methods for controlling Type I errors. Psychological Methods, 17(4), 679.
+
+    Note.
+    Generalized Hochberg is valid under MTP2 condition of the joint null 
+    distribution of the p-values
     
-    """
-    P-value adjustment procedure based on Holms method..
 
-    :param p: list of p-values to be adjusted
-    :return: pandas dataframe
-    """
+    :param p: list
+    The p-values to be adjusted.
+    
+    :param k: int
+    The  value for k-FWER. Defaults at to 1.
+    
+    :param proc: str
+    Indicates the method to be used. Choices are: ' Holm' ,'Hochberg'.
```

Comment:
Space in the Holm string should be removed

Reply:
Changed as per your suggestion.